question_5.rb

```diff
@@ -0,0 +1,23 @@
+# Taking IP address as input
+puts 'Enter IP Address: '
+ip = gets.chomp
+
+# Extracting the first octet
+# first -> takes the first segment 
+# to_i -> convert it to an integer 
+first_octet = ip.split('.').first.to_i
+
+# Determining the IP class
+if first_octet >= 1 && first_octet <= 126
```

Comment:
Please verify the logic, for ip address which start with 127 it will throw invalid ip error which is wrong.